operations/commit_queue.go

```diff
@@ -20,6 +20,8 @@ const (
 	itemFlagName        = "item"
 	pauseFlagName       = "pause"
 	resumeFlagName      = "resume"
+	commitFlagName      = "commit"
+	commitsFlagName     = "commits"
```

Comment:
I feel like the commit/commits api is confusing and we could just have a "commit" flag specified more than once...

Reply:
Switched to be just "--commits" or "-c"